packages/bff/src/model/domain/errors.ts

```diff
@@ -7,8 +7,8 @@ import {
 export const errorCodes = {
   purposeNotFound: "0001",
   missingClaim: "9990",
-  unknownTenantOrigin: "0011",
-  invalidJwtClaim: "0000", // Not used, change it if needed for a more specific error
+  tenantLoginNotAllowed: "0011",
+  tokenVerificationFailed: "0002",
```

Comment:
may I ask to sort them by code? there will be several error and it is easier to see duplicates